src/spring/azext_spring/app.py

```diff
@@ -560,5 +607,10 @@ def _fulfill_deployment_creation_options(skip_clone_settings, client, resource_g
             options.update(deployment_source_options_from_resource(active_deployment))
     if not options.get('sku', None):
         options['sku'] = get_spring_sku(client, resource_group, service)
+
+    if options['sku'].tier.upper() == 'STANDARDGEN2' and skip_clone_settings and kwargs['cpu'] is None and kwargs['memory'] is None:
```

Comment:
Could you please split this complex judgment statement into multiple variables with better readability?

Reply:
I have added a comment to explain the logic here